app/models/user.rb

```diff
@@ -3,6 +3,7 @@ class User < ApplicationRecord
   # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
 
   has_many :articles
+  has_many :comments, through: :articles
```

Comment:
Omit the through. Has many comments is suffice

Reply:
This is also because I was using this as reference:
https://guides.rubyonrails.org/association_basics.html
Will make changes because you told me to.

Reply:
Done